test/CommandLineUtils.Tests/ConstructorInjectionConventionTests.cs

```diff
@@ -79,7 +79,7 @@ public void ItPrefersIEnumOfOptionsFromUs()
         {
             var app = new CommandLineApplication<OptionsCtorCommand>();
             var services = new ServiceCollection().BuildServiceProvider();
-            app.Conventions.UseDefaultConventions().UseConstructorInjection(services);
+            app.Conventions.UseConstructorInjection(services).UseDefaultConventions();
```

Comment:
This is a symptom of the problem I mentioned above about initialization order. I think we need to make sure not to regress this as this will introduce subtle bugs for users.